make/conf/jib-profiles.js

```diff
@@ -1275,6 +1275,15 @@ var getJibProfilesDependencies = function (input, common) {
             ext: "tar.gz",
             revision: "3.4.2+1.0"
         },
+        tidy: {
+            organization: common.organization,
+            environment_name: "TIDY",
```

Comment:
Not sure this is a good idea. This will set an environment variable pointing to the "home_path" of the installation and will conflict with the configure arg below. The configure arg will win, so it will still work, but it could be confusing. A better environment variable name would be something like `TIDY_HOME`, but unless this is needed for something, I would just skip defining one.